sdk/src/program-manager.ts

```diff
@@ -580,6 +595,12 @@ class ProgramManager {
         let programName = options.programName;
         let imports = options.imports;
         let edition = options.edition;
+        let skipProof = options.skipProof;
+
+        // Skip proof defaults to false.
+        if (typeof skipProof === "undefined") {
+            skipProof = false;
+        }
```

Comment:
This can be condensed to the following

```suggestion
        let skipProof = options.skipProof ? options.skipProof : false;
```

Reply:
removed and created new methods for execution, deployment, and upgrade transactions specific to devnode.